### PR TITLE
feat: Add kafka cluster resource support

### PR DESCRIPTION
**Pull request by qboileau**
Create new `conduktor_kafka_cluster_v2` resource that support [Conduktor Console Kafka Clusters](https://docs.conduktor.io/platform/reference/resource-reference/console/#kafkacluster) and [Schema Registry](https://docs.conduktor.io/platform/reference/resource-reference/console/#schema-registry) configurations.

This PR add : 
- Resource terraform schema 
- Resource Console API models
- Mapping between models
- Terraform Resource implementation
- Resource tests 
- Resource Examples 
- Resource documentation

### CODE REVIEW

**Review by marnas — APPROVED** [review]
Great work

File: README.md [hunk 1]
```diff
@@ -43,11 +43,12 @@ This repository contains the Conduktor Terraform provider, which defines Condukt
 
 - [Console user](./docs/resources/user_v2.md)
 - [Console group](./docs/resources/group_v2.md)
+- [Kafka Clusters with Schema Regsitry](https://docs.conduktor.io/platform/reference/resource-reference/console/#kafkacluster)
```
Comment (Stu-conduktor) [inline README.md:46]:
```suggestion
- [Kafka Clusters with Schema Registry](./docs/resources/kafka_cluster_v2.md)
```

File: README.md [hunk 2]
```diff
@@ -37,17 +37,38 @@ This repository contains the Conduktor Terraform provider, which defines Condukt
...
+  - [Misc](#misc)
+- [Resources Roadmap](#resources-roadmap)
+- [Contributing](#contributing)
+- [License](#license)
 
 ## Supported resources
 
 - [Console user](./docs/resources/user_v2.md)
 - [Console group](./docs/resources/group_v2.md)
+- [Kafka Clusters with Schema Registry](https://docs.conduktor.io/platform/reference/resource-reference/console/#kafkacluster)
```
Comment (Stu-conduktor) [inline README.md:66]:
```suggestion
- [Kafka Clusters with Schema Registry](./docs/resources/kafka_cluster_v2.md)
```
```suggestion
- [Kafka Clusters with Schema Registry](https://docs.conduktor.io/platform/reference/resource-reference/console/#kafkacluster)
```
```suggestion
- [Kafka Clusters with Schema Registry](./docs/resources/kafka_cluster_v2.md)
```

Comment (Stu-conduktor) [inline README.md:66]:
```suggestion
- [Kafka Clusters with Schema Registry](./docs/resources/kafka_cluster_v2.md)
```

**Pull request merged**